src/Test/Perf/helloworld/HelloWorld.csx

```diff
@@ -0,0 +1,20 @@
+#load "../util.csx"
+using System.IO;
+
+var pathToCsc = Path.Combine(BinDebugDirectory(), "csc.exe");
```

Comment:
Good idea.  I'm probably going to rewrite the directory utilities in the next commit to use less compiler magic.  I'll fix this then.